src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -87,6 +88,22 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
         }
     }
 
+    public async createChildImpl(context: ICreateChildImplContext): Promise<PostgresDatabaseTreeItem> {
+        const client: PostgreSQLManagementClient = createAzureClient(this.root, PostgreSQLManagementClient);
+        const fullID: string = nonNullProp(this, 'fullId');
+        const resourceGroup: string = azureUtils.getResourceGroupFromId(fullID);
```

Comment:
You call `getResourceGroupFromId` quite a few times now. How about you add a `resourceGroup` property to this tree item class and set it once in the constructor